openmc/_utils.py

```diff
@@ -32,6 +34,11 @@ def download(url, checksum=None, as_browser=False, **kwargs):
     else:
         page = url
 
+    if output_path is not None:
+        cwd = Path.cwd()
+        Path(output_path).mkdir(parents=True, exist_ok=True) 
+        os.chdir(output_path)
```

Comment:
Rather than changing directory, I would just have open the file that is going to be written in the output directory. This would save you from having to change the directory back at the end.

Reply:
Thanks for taking a look. I shall do as you recommend with the removal of changing directory make the PR to the data repository.